Update getting started page

dipinknair opened the pull request:
- [x] Add grids
- [x] Update the version for embedded in linux
- [x] Move Installation guide to a separate page

Relates to #560 

**Current version**


![image](https://github.com/ansys/pymechanical/assets/26918585/9d54d5ee-aa9a-4d4e-bf01-996c5118ec1e)


**New version**


![image](https://github.com/ansys/pymechanical/assets/26918585/237e41c4-03ad-4ee2-b13a-b0305cc4ee6b)

RobPasMue commented:
I'd use a 3 column grid if possible rather than 2-1-2-1

RobPasMue commented on doc/source/getting_started/index.rst:
```diff
@@ -1,150 +1,74 @@
+
 ===============
-Getting started
+Getting Started
```
```suggestion
Getting started
```

Adhere to Google doc style

dipinknair commented:
New updates  @RobPasMue @koubaa 

- cards arranged in 2 columns
- Updated docs as per @sbradrick feedback.
- Fixed broken links
- added link check to make file. can be used with ``doc/make.bat linkcheck`` to see broken or redirected links in docs
- removed project index from main page
- renamed docs section titles

Revathyvenugopal162 commented on doc/source/examples/index.rst:
```diff
@@ -12,14 +12,12 @@ an embedded instance or remote session.
    .. grid-item-card:: Embedding Examples
       :link: https://embedding.examples.mechanical.docs.pyansys.com/index.html
       :text-align: center
-      :margin: 2 2 0 0
```
if you are removing the margin, i would recommend to add gutter for the cards. we follow 2 2 3 4 pattern as i wrote in some of the comments below.

dipinknair replied:
Reverting back to margin

Revathyvenugopal162 commented on doc/source/getting_started/index.rst:
```diff
@@ -1,150 +1,72 @@
 
-.. code::
+.. grid:: 2
+    :gutter: 2
```
```suggestion
    :gutter: 2 2 3 4
```

Revathyvenugopal162 commented on doc/source/getting_started/index.rst:
```diff
@@ -1,150 +1,72 @@
-PyMechanical from scratch on Windows and Linux for Python 3.9 through Python 3.12. You can install
-a wheelhouse archive on an isolated system with a fresh Python installation or on a
-virtual environment.
+This section provides the following information.
```
```suggestion
```

Revathyvenugopal162 commented on doc/source/getting_started/index.rst:
```diff
@@ -1,150 +1,72 @@
-Before running either, you must first verify that you can find
-the installed version of Mechanical using the ``ansys.tools.path`` package.
-This package is required to use PyMechanical.
+        Versioning requirements and support for Ansys Mechanical
```
```suggestion
        Versioning requirements and support for Ansys Mechanical.
```

Revathyvenugopal162 commented on doc/source/getting_started/index.rst:
```diff
@@ -1,150 +1,72 @@
-   >>> from ansys.tools.path import find_mechanical
-   >>> find_mechanical()
+.. grid:: 2
+    :gutter: 2
```
```suggestion
    :gutter: 2 2 3 4
```

Revathyvenugopal162 commented on doc/source/getting_started/index.rst:
```diff
@@ -1,150 +1,72 @@
+    :gutter: 2
 
-   or
+    .. grid-item-card:: :fas:`fa-solid fa-gears`  Launching PyMechanical
```
```suggestion
    .. grid-item-card:: :fas:`fa-solid fa-gears` Launching PyMechanical
```

Revathyvenugopal162 commented on doc/source/getting_started/index.rst:
```diff
@@ -1,150 +1,72 @@
 
-Verify your installation
-------------------------
+    .. grid-item-card:: :fas:`fa-solid fa-code-compare`  Versioning
```
```suggestion
    .. grid-item-card:: :fas:`fa-solid fa-code-compare` Versioning
```

Revathyvenugopal162 commented on doc/source/getting_started/index.rst:
```diff
@@ -1,150 +1,72 @@
+        :text-align: center
 
-   >>> find_mechanical(version=231)  # for specific version
+        Steps to run PyMechanical
```
```suggestion
        Steps to run PyMechanical.
```

Revathyvenugopal162 commented on doc/source/getting_started/index.rst:
```diff
@@ -1,150 +1,72 @@
-   >>> path = get_mechanical_path()
-   >>> print(path)
+.. grid:: 2
+    :gutter: 2
```
```suggestion
    :gutter: 2 2 3 4
```

Revathyvenugopal162 commented on doc/source/getting_started/index.rst:
```diff
@@ -1,150 +1,72 @@
-you can save this directory path using the ``save_mechanical_path`` function. Then use
-``get_mechanical_path`` and ``version_from_path`` functions to verify the path and version.
-For more details, refer to the :ref:`ref_ansys_tools_path_api`.
+        Steps for creating mechanical docker images
```
```suggestion
        Steps for creating mechanical docker images.
```

Revathyvenugopal162 commented on doc/source/getting_started/index.rst:
```diff
@@ -1,150 +1,72 @@
 
-   ('C:/Program Files/ANSYS Inc/v231/aisol/bin/winx64/AnsysWBU.exe', 23.1)  # windows
-   ('/usr/ansys_inc/v231/aisol/.workbench', 23.1) # Linux
+    .. grid-item-card:: :fab:`fa-brands fa-docker`  Docker setup  
```
```suggestion
    .. grid-item-card:: :fab:`fa-brands fa-docker` Docker setup  
```

Revathyvenugopal162 commented on doc/source/getting_started/index.rst:
```diff
@@ -1,150 +1,72 @@
-If you are on Windows with Python 3.9, unzip the ``ansys-mechanical-core-v0.11.dev0-wheelhouse-Windows-3.9``
-wheelhouse archive to a ``wheelhouse`` directory and then install it using ``pip`` like
-in the preceding example.
+        How to install and verify PyMechanical
```
```suggestion
        How to install and verify PyMechanical.
```

Revathyvenugopal162 commented on doc/source/getting_started/installation.rst:
```diff
@@ -0,0 +1,142 @@
+.. _ref_installation:
+==================
```
```suggestion

```

Revathyvenugopal162 commented on doc/source/getting_started/installation.rst:
```diff
@@ -0,0 +1,142 @@
+PyMechanical is compatible with Mechanical 2023 R1 and later on Windows
+and Linux. For more information please refer to  :ref:`ref_versioning` documentation.
+Later releases provide significantly better support and features.
+
```
```suggestion
```

Revathyvenugopal162 commented on doc/source/getting_started/installation.rst:
```diff
@@ -0,0 +1,142 @@
+
+   pip install ansys-mechanical-core
+
+
```
```suggestion
```

Revathyvenugopal162 commented on doc/source/getting_started/installation.rst:
```diff
@@ -0,0 +1,142 @@
+   pip install ansys-mechanical-core -f wheelhouse --no-index --upgrade --ignore-installed
+
+If you are on Windows with Python 3.9, unzip the ``ansys-mechanical-core-v0.11.dev0-wheelhouse-Windows-3.9``
+wheelhouse archive to a ``wheelhouse`` directory and then install it using ``pip`` like
```
```suggestion
wheelhouse archive to a ``wheelhouse`` directory and then install it using ``pip`` as
```

Revathyvenugopal162 commented on doc/source/getting_started/installation.rst:
```diff
@@ -0,0 +1,142 @@
+
+Verify your installation
+------------------------
+
```
```suggestion
```

Revathyvenugopal162 commented on doc/source/getting_started/installation.rst:
```diff
@@ -0,0 +1,142 @@
+
+Verify an embedded instance
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
```
```suggestion
```

Revathyvenugopal162 commented on doc/source/getting_started/installation.rst:
```diff
@@ -0,0 +1,142 @@
+.. code::
+
+    mechanical-env python
+
```
```suggestion
```

Revathyvenugopal162 commented on doc/source/getting_started/installation.rst:
```diff
@@ -0,0 +1,142 @@
+   If you are running on Linux, you must set some environment variables for
+   embedding of Mechanical in Python to work. A script that sets these variables is
+   available to install using pip:
+
```
```suggestion
```

Revathyvenugopal162 commented on doc/source/getting_started/running_mechanical.rst:
```diff
@@ -1,8 +1,8 @@
-***********************************************
-Using PyMechanical from a standard installation
-***********************************************
+======================
```
```suggestion
```

Revathyvenugopal162 commented on doc/source/getting_started/versioning.rst:
```diff
@@ -1,6 +1,8 @@
-**********
+.. _ref_versioning:
+
+==========
```
```suggestion
```

Revathyvenugopal162 commented on doc/source/getting_started/wsl.rst:
```diff
@@ -1,8 +1,8 @@
-PyAnsys libraries in the Windows Subsystem for Linux and Docker
-###############################################################
-
+===========================
```
```suggestion
```

Revathyvenugopal162 commented on doc/source/links.rst:
```diff
@@ -1,5 +1,5 @@
 .. #Pyansys
-.. _pyansys: https://docs.pyansys.com/dev/
+.. _pyansys: https://docs.pyansys.com/version/dev/
```
```suggestion
.. _pyansys: https://docs.pyansys.com/
```
default landing page is the dev version.

Revathyvenugopal162 reviewed (approved):
looks good some minor comments.

RobPasMue commented on doc/source/api/path.rst:
```diff
@@ -2,6 +2,7 @@
 Ansys path tools
 ================
+
 These methods on the ansys-tools-path module provide helper functions for configuration and discovery
```
```suggestion
These methods on the ``ansys-tools-path`` module provide helper functions for configuration and discovery
```

RobPasMue reviewed (approved):
LGTM! Great overall docs edit @dipinknair
